lib/xclarity_client/config_profile.rb

```diff
@@ -0,0 +1,19 @@
+module XClarityClient
+  class ConfigProfile
+    include XClarityClient::Resource
+
+    BASE_URI = '/profiles'.freeze
+    LIST_NAME = 'items'.freeze
+
+    attr_accessor :identifier, :items, :label, :profileName, :serverName, :uuid, :chassisName, :bayId,
+    :subBayId, :profileStatus, :templateName, :templateId, :type, :id, :externalId, :managementPatternPresent,
+    :addressPresent, :rackId, :unit, :ID, :name, :displayName, :displayId, :endPointType, :endPointId, 
```

Comment:
According to the REST API documentation, I think _rackId_ should be _rackID_, and remove the _ID_ attribute (I believe _id_ is used instead). Likewise, change _templateId_ to _templateID_.

Reply:
REST API doc for GET on /profiles/id says rackId
REST API doc for GET on /profiles says rackID
REST API doc for GET on /profiles/id  uses ID
REST API doc for GET on /profiles uses id
REST API doc for GET on /profiles/id uses templateId
REST API doc for GET on /profiles uses templateID
Hence, have retained all the 6 attributes.